mlflow/server/js/src/experiment-tracking/reducers/Reducers.js

```diff
@@ -34,6 +36,14 @@ import { SEARCH_MODEL_VERSIONS } from '../../model-registry/actions';
 import { getProtoField } from '../../model-registry/utils';
 import Utils from '../../common/utils/Utils';
 
+export const getExperimentSearchInput = (state) => {
+  return state.entities.searchInput.currentSearchInput;
```

Comment:
can we make sure that all introduced fields' names somehow indicate that they're tied to the experiments list search? otherwise, their function might be confused with e.g. runs list (where there is also a next page load feature, search inputs, similar flags etc.)